kubernetes/crd/domain-crd.yaml

```diff
@@ -10861,8 +10872,9 @@ spec:
                   will restart the required servers.
               introspectVersion:
                 type: string
-                description: If present, every time this value is updated, the operator
-                  will start introspect domain job
+                description: If the domain is running, the operator will restart its
+                  introspector job when this value is changed.This field is ignored
```

Comment:
Typo missing space after the period -> ". This field is ignored when the domainHomeSourceType source type is FromModel."